packages/librewolf-app/librewolf-app.pacscript

```diff
@@ -14,23 +14,19 @@ sha256sums_arm64=(
   "SKIP"
   "SKIP"
 )
-if [[ ${CARCH} == amd64 ]]; then
-  gnuarch="x86_64"
-else
-  gnuarch="aarch64"
-fi
 source=(
-  "https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/${pkgver}/LibreWolf.${gnuarch}.AppImage"
   "icon.png::https://gitlab.com/uploads/-/system/group/avatar/5407097/icon.png"
   "${gives}.desktop::https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/${pkgname}/${gives}.desktop"
 )
+source_amd64=("https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/${pkgver}/LibreWolf.x86_64.AppImage")
+source_arm64=("https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/${pkgver}/LibreWolf.aarch64.AppImage")
 breaks=("${gives}-deb" "${gives}-bin" "${gives}-git")
 repology=("project: librewolf")
 
 package() {
   cd "${_archive}"
   # Install appimage
-  install -Dm755 "LibreWolf.${gnuarch}.AppImage" "${pkgdir}/usr/bin/${gives}"
+  install -Dm755 LibreWolf.*.AppImage "${pkgdir}/usr/bin/${gives}"
```

Comment:
Same issue, don't glob, just move the check to inside of the function for this part